testnets/mantrachaintestnet/chain.json

```diff
@@ -0,0 +1,119 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "mantrachain",
```

Comment:
This value must now also be "mantrachaintestnet"

Reply:
Fixed.